patches/chromium/revert_roll_clang_llvmorg-14-init-1002-gb5e470aa-1.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@electronjs.org>
+Date: Mon, 23 Aug 2021 13:42:34 -0400
+Subject: Revert "Roll clang llvmorg-14-init-1002-gb5e470aa-1 :
+ llvmorg-14-init-1380-gee659383-1"
+
+This reverts commit 65172c258c45f5bc22f2018884d013b8d11f4b26.
```

Comment:
@jkleinsc is there a corresponding upstream issue ?

Reply:
I couldn't find one.